frontend/src/components/Metrics/IstioMetrics.tsx

```diff
@@ -401,7 +401,7 @@ class IstioMetricsComponent extends React.Component<Props, MetricsState> {
 
 const mapStateToProps = (state: KialiAppState) => {
   return {
-    jaegerIntegration: state.jaegerState.info ? state.jaegerState.info.integration : false,
+    jaegerIntegration: state.tracingState.info ? state.tracingState.info.integration : false,
```

Comment:
similar, if changed then it needs to also be changed in ReduxProps (which could then also be alphabetized :-))